jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -46,8 +46,8 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
-            H 3 * * * %TEST_MANIFEST=2.12.0/opensearch-2.12.0-concurrent-search-test.yml;BUILD_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.12.0/latest/linux/arm64/tar/builds/opensearch/manifest.yml
-            H 3 * * * %TEST_MANIFEST=2.12.0/opensearch-2.12.0-concurrent-search-test.yml;BUILD_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.12.0/latest/linux/x64/tar/builds/opensearch/manifest.yml
+            H */7 * * * %TEST_MANIFEST=2.12.0/opensearch-2.12.0-concurrent-search-test.yml;BUILD_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.12.0/latest/linux/arm64/tar/builds/opensearch/manifest.yml
+            H */7 * * * %TEST_MANIFEST=2.12.0/opensearch-2.12.0-concurrent-search-test.yml;BUILD_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.12.0/latest/linux/x64/tar/builds/opensearch/manifest.yml
```

Comment:
This is not required. Once a day is fine for concurrent search

Reply:
Updated.